ocaml/xapi/xapi_bond.ml

```diff
@@ -431,6 +431,21 @@ let create ~__context ~network ~members ~mAC ~mode ~properties =
             Db.PIF.set_disallow_unplug ~__context ~self:pif ~value:false)
           members
       end;
+
+      (* If the host has a cluster_host AND the cluster_host's PIF is a member, update the PIF *)
+      begin match Xapi_clustering.find_cluster_host ~__context ~host with
+      | Some cluster_host ->
+        let cluster_pif_ref = Db.Cluster_host.get_PIF ~__context ~self:cluster_host in
+        if List.mem cluster_pif_ref members
+        then begin
+          let master_pif_rec = Db.PIF.get_record ~__context ~self:master in
+          Xapi_clustering.assert_pif_prerequisites (master,master_pif_rec);
```

Comment:
It's a bit late to do asserts now, because the bond has already been created. Aborting here would leave things in a bad state. This is why there is a "Validate constraints" section in the beginning on the function.

However, do we really need this check? I thought that we concluded that the bond master meets the prerequisites by construction?

Reply:
Yeah, that's true, will remove them.